airgun/entities/activationkey.py

```diff
@@ -36,6 +36,12 @@ def read(self, entity_name, widget_names=None):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def get_repos(self, entity_name, repo_type="All", widget_names=None):
```

Comment:
widget_names is not unused

Reply:
The normal read does work here... However, you are not able to select ```repo_type``` when you call ```activationkey.read```. We are specifically testing that we can filter via custom repo_type. If we were to modify the normal read on activation keys, we may have to modify other tests out of this scope... Additionally, ```widget_names='repository_sets'``` is not the only thing we use ```activationkey.read``` for. In my mind, it does not make sense to modify read with ```repo_type``` when other widgets can also be called.

Reply:
I agree with @ColeHiggins2 on the purpose of `get_repos` function, but I would still remove `widget_names` as suggested by Omkar parameter as it is unused.

Reply:
this can be done easily without modifying the tests as well. 

```
def read(self, entity_name, widget_names=None, values=None):
        """Read all values for created activation key entity"""
        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
        view.fill(values)
        return view.read(widget_names=widget_names) 
```
where as the `values={"repository_sets.repo_type" : "All"}`  neverless, I'm okay with this change. You have to remove the `widget_names`

Reply:
non-blocking, please remove the unused `widget_names`